docs/networks/node-ops/evm-gateway/evm-gateway-setup.md

```diff
@@ -303,12 +303,53 @@ You will not be able to re-use the DB data dir from the previous versions.
 
 ### Account and Key Management
 
-EVM Gateway allows for Google and AWS Key Management Service (KMS) setup, which is the recommended way of setting up the gateway 
-for live networks. We recommend creating multiple KMS keys for the same Flow account (ideally 10 or more), how many depends on 
-the desired transaction throughput, since the keys are used in rotation when submitting the transactions. If too few keys 
-are configured it may result in sequence number collisions if the same key is used concurrently by multiple EVM client requests. 
+When operating an EVM Gateway it is important to understand how keys are configured and used. Each gateway instance 
+must be configured with a Flow account address, which is sufficiently funded and which it uses to pay to wrap EVM transactions into a 
+Cadence transaction when submitting to the Flow Access Node. This can be configured with a standalone private key file for 
+the `--coa-key` config flag. Alternatively, you may also use cloud KMS providers per the guidance below.
+
+#### COA Account signing key rotation
+
+The gateway implements a signing key rotation scheme to scale the use of the `COA_KEY` and enable it to be used across many
+EVM client transactions. This is configured by reusing the COA public key to create new on-chain signing keys. Although it may seem
+counter-intuitive to use the same COA public key when creating new signing keys, the newly added keys all occupy different 
+key slots on the account which enables the gateway to support concurrent transaction signing without causing nonce collisions 
+for EVM clients. 
+
+The following transaction creates 100 keys on the account signing the transaction to add these keys:
+```swift
+transaction {
+    prepare(signer: auth(AddKey) &Account) {
+		let firstKey = signer.keys.get(keyIndex: 0)!
+		let range: InclusiveRange<Int> = InclusiveRange(1, [100], step: 1)
+		for element in range {
+			signer.keys.add(
+				publicKey: firstKey.publicKey,
+				hashAlgorithm: HashAlgorithm.SHA2_256,
+				weight: 1000.0
+			)
+		}
+	}
+}
+```
+Signing keys which are added to the COA account are required to use the `SHA2_256` hashing algorithm.
```

Comment:
it doesn't have to be `SHA2_256`, it just has to match the hashing algorithm of the key that we're copying. For us at the foundation, it always has to be sha2 256, but that's not universally true

Reply:
Ah ok. Ardit had said that previously. Will update

Reply:
Cloud KMS Keys support only the `SHA2_256` hashing algorithm, that's why we used this one.